Ports/python3/package.sh

```diff
@@ -17,14 +17,16 @@ icon_file="../launcher.ico" # This is an older icon that's downloaded separately
 
 depends=("bzip2" "libffi" "libuuid" "ncurses" "openssl" "readline" "sqlite" "termcap" "zlib")
 
-configopts=("--enable-optimizations" "--disable-ipv6" "--without-ensurepip" "ac_cv_file__dev_ptmx=no" "ac_cv_file__dev_ptc=no")
+configopts=("--disable-ipv6" "--without-ensurepip" "ac_cv_file__dev_ptmx=no" "ac_cv_file__dev_ptc=no")
 
-export CC="${CC} --sysroot=${SERENITY_INSTALL_ROOT}"
 export BLDSHARED="${CC} -shared"
 
 pre_configure() {
     build="$("${workdir}/config.guess")"  # e.g. 'x86_64-pc-linux-gnu'
     configopts+=("--build=${build}")
+    if [ "$SERENITY_TOOLCHAIN" = "GCC" ]; then
+        configopts+=("--enable-optimizations")
+    fi
```

Comment:
Does `--enable-optimizations` have any effect when building with GCC? If not, leaving this here would be misleading to the reader. 

If I'm reading Python's `configure` correctly,  `--enable-optimizations` controls PGO and `-fno-semantic-interposition`. We are definitely not taking advantage of PGO with either of the two toolchains, and our GCC toolchain has `-fno-semantic-interposition` enabled by default.

Reply:
Ah yes, looking at the README.rst for 3.10.1 , --enable-optimizations currently simply enables PGO. LTO is enabled by a different flag (we could maybe try that one :thonk:).

We won't be able to take advantage of this flag unless we build on-target. I'll remove it completely instead.